python/lsst/resources/_resourcePath.py

```diff
@@ -85,11 +85,14 @@ class ResourcePath:
     root : `str` or `ResourcePath`, optional
```

Comment:
`uri` description above says "Can be scheme-less if referring to a local filesystem path", does it need to be updated to say scheme-less is relative to `root` (which does not have to be a local path)?